src/lib/i18n/locales/de-DE/translation.json

```diff
@@ -408,8 +408,8 @@
 	"Model not selected": "Modell nicht ausgewählt",
 	"Model Params": "Modell-Params",
 	"Model updated successfully": "Modell erfolgreich aktualisiert",
-	"Model Whitelisting": "Modell-Whitelisting",
-	"Model(s) Whitelisted": "Modell(e) auf der Whitelist",
+	"Model Allowlisting": "Modell-Allowlisting",
```

Comment:
"Modellauswahl einschränken" (Restrict model selection)

The text should describe the action that i trigger by toggling the toggle button. Should be the case for every other translation as well. Currently it's pretty misleading :)

<img width="586" alt="image" src="https://github.com/user-attachments/assets/dc9ce4dc-253b-499c-af9e-e5c013ada63a">